pkgs/development/python-modules/pytest-astropy-header/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-astropy-header";
+  version = "0.1.2";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1y87agr324p6x5gvhziymxjlw54pyn4gqnd49papbl941djpkp5g";
+  };
+
+  nativeBuildInputs = [ pytest ];
```

```suggestion
  propagatedBuildInputs = [ pytest ];
```
https://github.com/astropy/pytest-astropy-header/blob/d92f623f970a9ccf7ba3dfdba009c148a4ed77ba/setup.cfg#L22

---

Fixed in updated PR